internal/provider/provider_test.go

```diff
@@ -245,3 +304,47 @@ func testProviderTokenExists(testProvider *schema.Provider) resource.TestCheckFu
 		return nil
 	}
 }
+
+func createDefaultLdap(t *testing.T) *testdirectory.Directory {
+	td := testdirectory.Start(t,
+		testdirectory.WithDefaults(t, &testdirectory.Defaults{AllowAnonymousBind: true}),
+		testdirectory.WithNoTLS(t),
+	)
+
+	groups := []*gldap.Entry{
+		testdirectory.NewGroup(t, "admin", []string{"alice"}),
+		testdirectory.NewGroup(t, "admin", []string{"eve"}, testdirectory.WithDefaults(t, &testdirectory.Defaults{UPNDomain: "example.com"})),
```

Comment:
consider removing this line: I don't think, you're using "eve" and the group is a dup anywho.